src/main/java/com/anusha/collectiontasks/AN007_StringSplitUsingDelimiter.java

```diff
@@ -0,0 +1,12 @@
+package com.anusha.collectiontasks;
+
+public class AN007_StringSplitUsingDelimiter {
+public static void main(String[] args){
+    String string = "Goutham-K-Rao";
+    String[] parts = string.split("-");
+
+    for(int i=0;i<parts.length;i++){
+    System.out.println(parts[i]);
```

Comment:
can you look at utilising matchers to check if all the words are present in the array